### PR TITLE
WIP on transforming to MDN-like model of documentation of MapML.

### PR DESCRIPTION
**Pull request by prushforth**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -3,26 +3,16 @@ module.exports = {
...
-      {
-        'Layers': [
-          'layers/layer',
-          'layers/static-tiles',
-          'layers/static-features',
-          'layers/static-images',
-          'layers/templated-tiles',
-          'layers/templated-features',
-          'layers/templated-images',
-        ],
```
Comment (ahmadayubi) [inline sidebars.js:19]: Looks good so far! I was wondering how you plan on presenting the different layer types as I see they are removed from the menu in the changes.

Comment (prushforth) [reply sidebars.js:19]: I was thinking of example pages or tutorial pages.

Comment (ahmadayubi) [reply sidebars.js:19]: Ahh okay, that's a good way to organize it.

File: docs/elements/meta.md [hunk 2]
```diff
@@ -12,48 +12,69 @@ maps.
...
-Description of attribute (repeat as required)
+The `name` attribute identifies the type of metadata that is being set. Possible
+values of `name` related to maps include:
 
 | Name          | Description                                          	  |
 |--------------	|--------------------------------------------------------	|
+| projection    | Set the projection for the document |
+| extent        | Define the bounds of the document in terms of corner position keywords and axis names, potentially combined with a zoom value if necessary (depending on coordinate system). |
+| cs            | Identifies the default coordinate reference system of coordinate strings found in the document. |
+| zoom          | Specify the 'native' zoom range min, max and value for displaying the document |
```
Comment (prushforth) [inline docs/elements/meta.md:25]: @ahmadayubi Is it correct to describe the min/max/value of `<map-meta name="zoom" ...>` as "native??  I am thinking that the "value" value might be native, but the min/max values are the client side zoom range /bounds to display the content in, so not the "native" (server) zoom.  If you have a templated tile link, for example, the min/max on the inputs are "native" though.  amirite???

**Pull request merged**